js/main.js

```diff
@@ -1,105 +1,105 @@
   $loading.className = '';
-  var xhrMonth = new XMLHttpRequest();
+  const xhrMonth = new XMLHttpRequest();
   xhrMonth.open('GET', 'https://lfz-cors.herokuapp.com/?url=' + targetUrl + year + '/' + month);
   xhrMonth.setRequestHeader('token', 'abc123');
   xhrMonth.responseType = 'json';
   xhrMonth.addEventListener('error', function () {
-    var $failToLoadPage = document.querySelector('.failed-page');
+    const $failToLoadPage = document.querySelector('.failed-page');
     $failToLoadPage.className = 'failed-page';
   });
   xhrMonth.addEventListener('load', renderMonth);
   xhrMonth.send();
   return xhrMonth;
 }
 
-var xhrMonth = getCalendarData(dateMonthNum);
+let xhrMonth = getCalendarData(dateMonthNum);
 
-function viewSwap(event) {
-  if ($calendarPage.className === 'hidden' && event.target.className === 'fas fa-arrow-left') {
-    $journalPage.innerHTML = '';
-    $rightarrow.className = 'fas fa-arrow-right';
-    $calendarPage.className = 'container background-color rel margin-top padding-bottom';
-    data.editing = null;
-    return;
-  }
-  if ($calendarPage.className !== 'hidden' && event.target.className === 'fas fa-arrow-left') {
-    currentMonthNum -= 1;
-    if (currentMonthNum < 1) {
-      year -= 1;
-      currentMonthNum = 12;
-    }
-    xhrMonth = getCalendarData(currentMonthNum);
-  }
-  if ($calendarPage.className !== 'hidden' && event.target.className === 'fas fa-arrow-right') {
-    currentMonthNum += 1;
-    if (currentMonthNum > 12) {
-      currentMonthNum = 1;
-      year += 1;
-    }
-    xhrMonth = getCalendarData(currentMonthNum);
-  }
-}
-
-function addPhoto(event) {
-  if (event.target.name !== 'photoURL') {
-    return;
+function getCalendarMonth(numericDate) {
+  const monthDate = numericDate.split('-');
+  let currentMonth = '';
+  if (monthDate[1] === '01') {
+    currentMonth = 'January';
+    currentMonthNum = 1;
+  } else if (monthDate[1] === '02') {
+    currentMonth = 'February';
+    currentMonthNum = 2;
+  } else if (monthDate[1] === '03') {
+    currentMonth = 'March';
+    currentMonthNum = 3;
+  } else if (monthDate[1] === '04') {
+    currentMonth = 'April';
+    currentMonthNum = 4;
+  } else if (monthDate[1] === '05') {
+    currentMonth = 'May';
+    currentMonthNum = 5;
+  } else if (monthDate[1] === '06') {
+    currentMonth = 'June';
+    currentMonthNum = 6;
+  } else if (monthDate[1] === '07') {
+    currentMonth = 'July';
+    currentMonthNum = 7;
+  } else if (monthDate[1] === '08') {
+    currentMonth = 'August';
+    currentMonthNum = 8;
+  } else if (monthDate[1] === '09') {
+    currentMonth = 'September';
+    currentMonthNum = 9;
+  } else if (monthDate[1] === '10') {
+    currentMonth = 'October';
+    currentMonthNum = 10;
+  } else if (monthDate[1] === '11') {
+    currentMonth = 'November';
+    currentMonthNum = 11;
+  } else if (monthDate[1] === '12') {
+    currentMonth = 'December';
+    currentMonthNum = 12;
```

Comment:
I think there is a better way to do this. You can use any string as a key to get a value from an object.

```js
const months = {
  '01': 'January',
  '02': 'February',
  // ...
}
const monthName = months[monthDate[1]);
const monthNumber = parseInt(monthDate[1]);

```